common/xml/projects.xml

```diff
@@ -0,0 +1,214 @@
+			<sourceDesc>
+				<p>This is the source.</p>
+			</sourceDesc>
+		</fileDesc>
+		<profileDesc>
+			<langUsage>
+				<language ident="en"/>
+			</langUsage>
+			<!-- the textClass element can be lodged in a separate XML file and included here with XInclude if it gets unwieldy -->
+		</profileDesc>
+		<revisionDesc>
+			<change when="2022-10-25" who="#jhf">Created file</change>
+			<change when="2022-10-30" who="#brg">Added list of projects to test xml:id naming
+				convention</change>
+		</revisionDesc>
+	</teiHeader>
+	<text xml:lang="en">
+		<front>
+			<div>
+				<head>About the DHQ Project Registry</head>
+				<p>This is an initial, local registry of digital humanities projects that are
+					referenced from within DHQ articles. DHQ is working in partnership with other
+					publications (in particular, Reviews in Digital Humanities) to develop a broader
+					and more public project registry, possibly using Wikidata or other public linked
+					open data platforms, to which this registry will refer. Currently this registry
+					exists simply to create a target for project references and enable
+					identification and disambiguation.</p>
+			</div>
+			<div>
+				<head>Usage and expansion</head>
+				<p>Each DHQ article contains in its metadata one or more project identifiers which
+					in turn reference items in this registry. These project identifiers will be used
+					in several ways within the DHQ publishing system: <list>
+						<item>in the DHQ search interface, to enable readers to search for articles
+							that reference or discuss a specific project</item>
+						<item>to generate a project index that groups together articles based on
+							project references</item>
+						<item>to support analytics and visualizations</item>
+					</list>
+				</p>
+				<p>When an article is accepted for publication, as part of the encoding process the
+					managing editors will encode an inventory of project references in the article
+					metadata. Any new projects will be added to the registry.</p>
+			</div>
+			<div>
+				<head>Definition of <term>digital humanities project</term></head>
+				<p>Criteria for characterizing something as a digital humanities project: 
+						<list>
+							<item>goal that goes beyond a singular output</item>
+							<item>an organization that continues after the output has been achieved</item>
+							<item>typically (but not necessarily) more than one person</item>
+							<item>an organization, not solely a publication</item>
+						</list>
+				</p>
+			</div>
+		</front>
+		<body>
+			<listOrg>
+				<!-- use @cert with value "low" in cases where we're not sure whether the entity is actually a "project" according to our definition -->
+				<org xml:id="" cert="['low' if not certain]">
+					<name>[project name here]</name>
+					<ptr type="project_site" target=""/>
+				</org>
+			</listOrg>
+
+			<listOrg>
+				<org xml:id="">
+					<name>Lincoln/Net</name>
+					<!-- article 000003 -->
```

Comment:
All 11 `<org>`s other than the 1st have a comment that gives the article number, I presume of the article where this organization is mentioned. Is this information supposed to be machine-actionable, or just for humans? (And if not machine-actionable, why not?) Could there be more than one? Could there be lots?
If we want this to be machine-actionable, an unvalidatable comment is not the way to go. My instinct is that if there will always be 1 and only 1 article, or at most a handful of them, to use the `@source` of `<org>`:
~~~xml
        <org xml:id="xmp1" source="../../articles/000003/000003.xml">
~~~
Other possibilities include using a `<ptr type="found_in" target="…"/>`, a `<?dhq extracted from … ?>` PI, or an `<idno type="source_article">…</idno>` inside the `<org>`. (Note that if you do not like writing out an entire relative path like that, a `<prefixDef>` could be used to shorten it to `art:000003` or some such.)